### DISCUSSION TITLE
Want to make 2 notifiction send which depends on each other

### DISCUSSION THREAD

**Discussion by AashirRaz** [Windos/BurntToast#227] [Q&A]
Hey guys, need help i want to show a notification which contains a title and a body and two buttons named View in Folder, and send to skype, if any body click on View in folder i have a desired path where it should open folder , and if i click on send to skype button a new notification should be triggered having a text a box and a send button , then i we should for now i want it to just say sending , i will write the logic for sending just help me with this feature

**Comment by Windos** [comment]
The path you'll want to go down to achieve this is "actionable" notifications. I wrote about this on my blog here: https://toastit.dev/2020/08/07/actionable/

At the moment, that requires PowerShell 7.1 to work, though v1 of BurntToast will have it working back in Windows PowerShell and PowerShell 6.

Conceptually, what you're doing is spawning a new toast within the event action of the previous toast.

For "just the code" check out: https://github.com/Windos/Talks/blob/main/2021/02%20-%20February/23%20-%20DuPSUG%20-%20Croque%20Monsieur/Demos/09_SelectionBox.ps1

**Reply by AashirRaz** [reply]
i have able to invoke buttons , i have a new problem if you help 
--------------------------------------------x------------------------------------------------x------------------------------------
param (
    [string]$pathToFile,
    [string]$message
)

Write-Host "-------------> From Send To Whom $pathToFile $message"

$Activated = {
    Write-Host $using:pathToFile
    Write-Host "Activated (Arguments: $($Event.SourceArgs[1].Arguments))"
    
    if($Event.SourceArgs[1].Arguments -eq "jabba") {
        Write-Host "Invoking sendToSkype.ps1 script"
        $output = Invoke-Expression "python C:\AashirRaza\Scripts\sendToSkype.py 'rijab raza' $pathToFile $message"
        Write-Host $output
        Write-Host $Event.SourceArgs[1].UserInput.value
        return
    }  else {
        Write-Host "Activated"
    }
    $Global:ToastEvent = $Event
}

$Dismissed = {
    Write-Host "Dismissed (Reason: $($Event.SourceArgs[1].Reason)"
    if ($Event.SourceArgs[1].Reason -eq "userCanceled")  {
        Write-Host "User Canceled"
    } elseif ($Event.SourceArgs[1].Reason -eq "timedOut") {
        Write-Host "Timed Out"
        $output = Invoke-Expression "python C:\AashirRaza\Scripts\sendToSkype.py 'rijab raza' 'C:\profile.jpg' 'This is a test message from Javris to Skype from activated'"
    } else {
        Write-Host "Unknown Reason"
    }
    $Global:ToastEvent = $Event
}

Write-Host "handleSendSkypeNotification"

$SendToSkype = New-BTButton -Content 'Send to Skype' -Id 'Reply001' -Arguments "jabba" -ActivationType Background

$a = New-BTAction -Buttons $SendToSkype -Inputs (New-BTInput -Id Reply001 -PlaceholderContent 'Type a reply')

$Text1 = New-BTText -Content "Send to Skype"
$Text2` = New-BTText -Content "Who do you want to send this to?"
$Image1 = New-BTImage -Source "C:\profile.jpg" -AppLogoOverride -Crop Circle

$Binding1 = New-BTBinding -Children $Text1, $Text2 -AppLogoOverride $Image1
$Visual1 = New-BTVisual -BindingGeneric $Binding1

$Content1 = New-BTContent -Visual $Visual1 -Actions $a -Duration Long

Submit-BTNotification -Content  $Content1 -ActivatedAction $Activated -DismissedAction $Dismissed
----------------------------------------x-------------------------------------x-------------------------------------------------x

This is is my file of ps1 i am getting the param in the first Write-Host method but when i'm trying to access these param in $Activated block of code they are empty , can you tell me why and what would be the possible solution for it

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
